web/src/main/resources/org/openmrs/web/filter/footer.vm

```diff
@@ -1,10 +1,10 @@
 <footer class="app-footer">
     <div class="footer-content">
         <div class="footer-logo">
-            <img src="images/openmrs_logo_text.gif" alt="OpenMRS Logo" class="logo-image" />
+            <img src="images/openmrslogo.png" alt="OpenMRS Logo" class="logo-image" />
```

Comment:
Where is this image stored?

Reply:
looking into why git missed it

Reply:
Did you forget this?

Reply:
it was added https://github.com/openmrs/openmrs-core/pull/5025/files#diff-99d4b9849789c3c81954bf25d0501dbcec3ce7f4739b61e6b9b2ec4d94c72788